IzvorniKod/wwwroot/css/chat.css

```diff
@@ -0,0 +1 @@
+﻿
```

Comment:
Jeli ovo samo prazan css file, ako je jeli potreban pa je tu ili neki drugi razlog?

Reply:
Taj file je tu da ga frontend tim može editati kada uređivaju tu stranicu.

Reply:
Thinking about the future that's what makes you a good emply,
okej np resolved